client/src/views/Game.vue

```diff
@@ -35,11 +35,18 @@
             class="pa-3 player-drawer-item"
             two-line
           >
-            <v-list-item-icon>
-              <v-icon class="pt-3">
-              mdi-account
-              </v-icon>
-            </v-list-item-icon>
+          
+            <div class="col">
+              <v-list-item-icon class="ml-n4">
+                <v-icon class="pt-3">
+                mdi-account
+                </v-icon>
+              </v-list-item-icon>
+              
+              <!-- Number under the icon when pane is collapsed -->
+              <div v-if="player.cards != undefined" class="row ml-n3">{{ player.cards.length }}</div>
```

Comment:
From a UX perspective. @Aaron-Fine and I were trying to avoid a number as to actually build an `uno-like` experience of having to count cards.

So generating that somewhat takes away from that moment.

However, I'm not immediately opposed to this change, it just is opposite what we had intended.

In terms of actual design, can you include a screenshot of what the changes will look like on the PR? Also, I'm not seeing anything about the pane being collapsed/not collapsed. Were you intending this to just show up when it was collapsed or does it show all the time?